src/app.js

```diff
@@ -1 +1,28 @@
 'use strict';
+
```

Comment:
```suggestion
/* eslint-disable no-console */
```
If the console statements are required for the task, consider disabling the linter rule to fulfill the task requirements and pass all tests